hca/dss/util/__init__.py

```diff
@@ -51,3 +51,7 @@ def hardlink(source, link_name):
         # Check device first because different drives can have the same inode number
         if source_stat.st_dev != dest_stat.st_dev or source_stat.st_ino != dest_stat.st_ino:
             raise
+    except Exception as e:
```

Comment:
Let's discuss. I'm worried that a systematic error causes all files to be copied.

Reply:
Can you please reach out to me about this?

Reply:
@jessebrennan and I talked and agreed to only fall back to aking a copy on EMLINK and to fail hard on all other errors.